docs/workflows/index.md

```diff
@@ -0,0 +1,15 @@
+---
+myst:
+  html_meta:
+    "description lang=en": |
+      Workflows showcasing different SED loaders and methods
+---
+# Workflows
+
+```{toctree}
+:caption: Basic concepts
+../tutorial/2_conversion_pipeline_for_example_time-resolved_ARPES_data
+../tutorial/4_hextof_workflow
+../tutorial/6_binning_with_time-stamped_data
+../tutorial/7_correcting_orthorhombic_symmetry
```

Comment:
This separation does not really make sense to me.

Reply:
I think it makes sense to have an introduction to the different aspects of the library, irrespective of the different loaders/workflows at different instruments.
What is your suggestion?

Reply:
I reordered things slightly, and added a section advanced topics